src/test/java/com/fasterxml/jackson/core/json/async/AsyncConcurrencyTest.java

```diff
@@ -123,24 +130,20 @@ void _testConcurrentAsyncOnce(final int round, final int maxRounds) throws Excep
         final int REP_COUNT = 99000;
         ArrayList<Future<?>> futures = new ArrayList<Future<?>>();
         for (int i = 0; i < REP_COUNT; i++) {
-            Callable<Void> c = new Callable<Void>() {
-                @Override
-                public Void call() throws Exception {
-                    WorkUnit w = q.take();
-                    try {
-                        if (w.process()) {
-                            completedCount.incrementAndGet();
-                        }
-                    } catch (Throwable t) {
-                        if (errorCount.getAndIncrement() == 0) {
-                            errorRef.set(t.toString());
-                        }
-                    } finally {
-                        q.add(w);
+            Callable<Void> c = () -> {
```

Comment:
It'd be great to avoid changes like this to reduce size of diffs.... I assume this may have been introduced by IDE so hard to avoid.